Start9Labs/synapse-wrapper
# Add Synapse-Admin (#51)

* Update backups to use new eOS APIs
* latest synapse upstream version
* Add vps build
* Introducing Synapse-Admin
* persist config and timing bump
* multiple config changes (#50)
* disable federation health check if federation disabled
* add back privoxy settings
* persist config and timing bump
* singel equals
* nginx config fix
* Add admin interface
* disable federation port
* fix nginx creation
* secure synapse and seperate admin
* clean for vps tar image
* release-notes update
* config fix
* only client and federation
* fix email
* fix health check, better naming, better instructions
* timeout adjustments and fresh install fix

---------
